library/src/scala/quoted/Quotes.scala

```diff
@@ -2621,6 +2624,7 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
 
     /** Methods of the module object `val GivenSelector` */
     trait GivenSelectorModule { this: GivenSelector.type =>
+      @experimental def apply(bound: Option[TypeTree]): GivenSelector
```

Comment:
```suggestion
      @experimental def apply(bound: Option[TypeRepr]): GivenSelector
```

I would suggest internally wrapping the type in TypeTree with the default span

Reply:
That would be inconsistent with `unapply` and the `.bound` accessor method. Is that a problem?

Reply:
`TypeTree` makes more sense to me too